Libraries/LibWeb/WebGL/OpenGLContext.cpp

```diff
@@ -252,7 +268,7 @@ void OpenGLContext::allocate_painting_surface_if_needed()
 
     EGLint texture_target_name = 0;
 
-#if defined(AK_OS_MACOS)
+#    if defined(AK_OS_MACOS)
```

Comment:
this indentation change looks suspicious. I think it was fine and we don't need these extra spaces?